backend/app/api/routes/inquiries.py

```diff
@@ -24,3 +27,30 @@ def create_inquiry(*, session: SessionDep, inquiry_in: InquiryCreate) -> Inquiry
     session.commit()
     session.refresh(inquiry)
     return inquiry
+
+
+@router.get("/", response_model=InquriesPublic)
+def read_inquries(
+    session: SessionDep, skip: int = 0, limit: int = 100
+) -> InquriesPublic:
+    """
+    Retrieve inquries.
+    """
+    count_statement = select(func.count()).select_from(Inquiry)
+    count = session.exec(count_statement).one()
+
+    statement = select(Inquiry).offset(skip).limit(limit)
+    inquries = session.exec(statement).all()
```

Comment:
Is direct DB access from the controller fine, or should we stick to the CRUD service pattern?

Reply:
I think we'll eventually migrate our codebase to use CRUD service pattern.
Our proposed [backend structure documentation ](https://docs.google.com/document/d/1EWSyKXvgutzK7QA8PH6iQgnl0_7q4QQpmqjSTBY24Vg/edit?usp=sharing).

Reply:
Something like this:

```
├── app  # Contains the main application files.
│   ├── __init__.py   # This file makes "app" a "Python package".
│   ├── main.py       # Initializes the FastAPI application.
│   ├── api
│   │   ├── main.py  # Combines submodule routers.
│   │   ├── routes
│   │   │   ├── __init__.py
│   │   │   ├── users.py  # Defines routes and endpoints related to users.
│   │   │   ├── items.py  # Defines routes and endpoints related to items.
│   │   │   └── login.py  # Defines routes and endpoints related to login.
│   ├── crud 
│   │   ├── __init__.py 
│   │   ├── items.py  # Defines CRUD operations for items.
│   │   └── user.py  # Defines CRUD operations for users.
│   ├── models
│   │   ├── __init__.py
│   │   ├── item.py  # Defines database models for items.
│   │   └── user.py  # Defines database models for users.
│   ├── services  # New directory for business logic and service layer.
│   │   ├── __init__.py
│   │   ├── items.py  # Service layer for item-related business logic.
│   │   └── user.py  # Service layer for user-related business logic.
│   ├── alembic # Database migrations.
│   ├── core # Database setup, configuration, Pydantic settings, docs.
│   ├── tests 
│   │   ├── api/routes/
│   │   ├── crud/
│   │   └── services/  # New directory for service layer tests.

```

Reply:
But you should be making the calls through crud.py which is still in the repo